nebula_ros/src/continental/continental_ars548_decoder_wrapper.cpp

```diff
@@ -69,28 +66,36 @@ ContinentalARS548DecoderWrapper::ContinentalARS548DecoderWrapper(
 
   detection_pointcloud_pub_ = parent_node->create_publisher<sensor_msgs::msg::PointCloud2>(
     "detection_points", rclcpp::SensorDataQoS());
-  object_pointcloud_pub_ =
-    parent_node->create_publisher<sensor_msgs::msg::PointCloud2>("object_points", pointcloud_qos);
+  object_pointcloud_pub_ = parent_node->create_publisher<sensor_msgs::msg::PointCloud2>(
+    "object_points", rclcpp::SensorDataQoS());
+
+  autoware_objects_pub_ = parent_node->create_publisher<autoware_sensing_msgs::msg::RadarObjects>(
+    "radar_objects", rclcpp::SensorDataQoS());
 
   scan_raw_pub_ =
-    parent_node->create_publisher<radar_msgs::msg::RadarScan>("scan_raw", pointcloud_qos);
+    parent_node->create_publisher<radar_msgs::msg::RadarScan>("scan_raw", rclcpp::SensorDataQoS());
 
-  objects_raw_pub_ =
-    parent_node->create_publisher<radar_msgs::msg::RadarTracks>("objects_raw", pointcloud_qos);
+  objects_raw_pub_ = parent_node->create_publisher<radar_msgs::msg::RadarTracks>(
+    "objects_raw", rclcpp::SensorDataQoS());
 
   objects_markers_pub_ =
     parent_node->create_publisher<visualization_msgs::msg::MarkerArray>("marker_array", 10);
 
   diagnostics_pub_ =
     parent_node->create_publisher<diagnostic_msgs::msg::DiagnosticArray>("diagnostics", 10);
 
+  radar_info_pub_ = parent_node->create_publisher<autoware_sensing_msgs::msg::RadarInfo>(
+    "radar_info", rclcpp::SensorDataQoS());
```

Comment:
Semantically speaking, I think [`transient local` durability](https://docs.ros.org/en/humble/Concepts/Intermediate/About-Quality-of-Service-Settings.html#:~:text=Transient%20local%3A%20the%20publisher%20becomes%20responsible%20for%20persisting%20samples%20for%20%E2%80%9Clate%2Djoining%E2%80%9D%20subscriptions.) would make sense here.

Performance-wise, this would allow subscribers that join later to immediately receive radar info instead of having to wait for up to a second.

Reply:
ROS in general uses SensorDataQoS for sensor related data, and having mixed QoS in my experience causes users to make mistakes. What do you think @drwnz ?

Reply:
I think this is something we can review later when we do the LIDAR info topics. For now, let's keep it simple and run the same QoS as the data topics.